tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "moduleResolution": "node",
     "outDir": "dist/",
     "strict": true,
-    "target": "es2018"
+    "target": "es2018",
+    "lib": ["dom", "es6", "es2019"]
```

Comment:
Added that to be able to use `Object.fromEntries()`

Reply:
Object.fromEntries was added in `ECMAScript 2019`. Do we need to include `dom` and `es6 a.k.a ECMAScript 2015` too?

Reply:
This [SO question](https://stackoverflow.com/questions/63943629/what-is-the-typescript-compilers-default-lib-value) misled me a bit (the `es6` is not imported part) but looking at Typescript source it seems that `es2019` also import previous libs and `dom`, so it is indeed not necessary